src/System.Runtime.Extensions/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,7 +1,4 @@
 Compat issues with assembly System.Runtime.Extensions:
 MembersMustExist : Member 'System.AppDomain.GetThreadPrincipal()' does not exist in the reference but it does exist in the implementation.
 MembersMustExist : Member 'System.Environment.FailFast(System.String, System.Exception, System.String)' does not exist in the reference but it does exist in the implementation.
```

Comment:
This overload of FailFast I believe was never API reviewed so it's correct that it's not in the ref.